packages/okta-angular/README.md

```diff
@@ -144,6 +171,86 @@ const oktaConfig = {
 };
 ```
 
+## Reference
```

Comment:
I think this reference section needs another pass.  Right now it appears to be serving multiple purposes:

- Show you all the available classes you can import
- Tells you about some configuration options (but doesn't tell you where they go or how to use them)
- Shows you how to do configure the final redirect URL for login and logout operations
- Documents some, but not all, of the public methods available on the `OktaAuthModule` service

The content in here is good, though I feel that it needs to be moved around to other sections, and that this "reference" section should be more focused on describing the `OktaAuthModule` and what it provides you.